plugins/out_stackdriver/stackdriver.c

```diff
@@ -543,31 +659,51 @@ static int stackdriver_format(struct flb_config *config,
         /* Get timestamp */
         flb_time_pop_from_msgpack(&tms, &result, &obj);
 
+        num_pack = 3;
         /*
          * Pack entry
          *
          * {
+         *  "labels": "...",
          *  "logName": "...",
          *  "jsonPayload": {...},
          *  "timestamp": "..."
          * }
          */
+        labels_ptr = parse_labels(ctx, obj);
+        if (labels_ptr != NULL) {
+            num_pack += 1;
```

Comment:
Perfectly fine, though ++ would be more traditional. Optional.

Reply:
thanks.